roles/infrastructure/defaults/main.yml

```diff
@@ -18,6 +18,19 @@ infrastructure_vnet_subnets:
       - serviceName: Microsoft.DBforPostgreSQL/flexibleServers
         name: subnet-{{ deployment_id }}-postgres-delegation
 
+# Database
+infrastructure_database_server_user: "user_placeholder"
+infrastructure_database_server_password: "test1234!"
```

Comment:
Do you think these should be required vars? Just so people don't end up using bad creds automatically by not reading the readme.